tests/test.ts

```diff
@@ -1,147 +1,187 @@
-import util from 'util';
-import test from 'ava';
-import {jsdom} from 'jsdom';
-import m from '.';
+import * as util from 'util';
```

Comment:
I'd rather put the `tests` directory into the `source` directory. This way, everything regarding code sits together. This also means that the `index.ts` file is compiled to `dist/index.js` which seems a little better than `dist/src/index.js`.

The only thing that should be done then is in the `prepublish` hook. Install [`del-cli`](https://github.com/sindresorhus/del-cli) as dev dependency and set this as prepublish script.
```
npm run build && del dist/tests
```

Reply:
Agreed